packages/client-app/src/components/DataTable/DataTableCell.vue

```diff
@@ -11,7 +11,7 @@ withDefaults(
 <template>
   <component
     :is="is"
-    class="h-8 min-w-0 border-b-1/2 border-r-1/2 border-r-[inherit] text-sm last:border-r-0 group-last:border-b-transparent w-fit min-w-full has-[input[type=checkbox]]:w-8 has-[input[type=checkbox]]:min-w-8"
+    class="h-8 border-b-1/2 border-r-1/2 text-sm last:border-r-0 group-last:border-b-transparent"
```

Comment:
does this maintain square on checkbox cell + hover?

Reply:
Seems to? 

<img width="722" alt="Google Chrome-2024-06-19-16-30-26@2x" src="https://github.com/scalar/scalar/assets/6374090/e4c7dca9-7862-4a91-b0f1-72f572d0eea5">

Reply:
it's not well squared anymore, will add it back!